util/dateutil/util.go

```diff
@@ -10,8 +10,8 @@ import (
 const (
 	shortDateIdLayout   = "2006-01-02"
 	dateIdLayout        = "2006-01-02-15-04-05Z-0700"
-	shortDateNameLayout = "02 Jan 2006"
-	dateNameLayout      = "02 Jan 2006 15:04"
+	shortDateNameLayout = "Mon, Jan 02, 2006"
```

Comment:
i guess you should not revert the date string format? As we agreed to change it

Reply:
That's brand new prod requirement!
https://linear.app/anytype/issue/GO-4590/support-the-date-format-once-again
Extra comma added